provider/bidengine/order.go

```diff
@@ -266,7 +265,7 @@ func (o *order) shouldBid(group *dtypes.Group) bool {
 		return false
 	}
 
-	if err := validation.ValidateDeploymentGroup(group.GroupSpec); err != nil {
+	if err := group.GroupSpec.ValidateBasic(); err != nil {
```

Comment:
cool.  I think that this might not be needed anymore since everything has the same values, but what the heck.